test/puppetlabs/puppetdb/http/reports_test.clj

```diff
@@ -45,20 +46,18 @@
    [:start-time :end-time]
    ;; the response won't include individual events, so we need to pluck those
    ;; out of the example report object before comparison
-   (dissoc report :resource-events)))
+   (update-in report [:resource-events] (comp keywordize-keys munge-resource-events))))
 
 (defn reports-response
   [version reports]
-  (set (map (case version
-              :v3 (comp #(remove-status % :v3) #(remove-environment % :v3) report-response)
-              report-response) reports)))
+  (set (map report-response reports)))
 
-(defn remove-receive-times
+(defn treat-reports-response
```

Comment:
Can you come up with a more descriptive function name here? Maybe munge-for-compare, or prep-for-compare or something like that?